django_project/social_users/templates/social_users/profile.html

```diff
@@ -23,7 +32,7 @@
                         {% endif %}
                     {% endfor %}
                 {% endif %}
-                {% if user.profile_picture == "" and not user.social_auth.get.extra_data.avatar %}
+                {% if not user.profile_picture and not user.social_auth.get.extra_data.avatar %}
```

Comment:
i think this is fixed, please pull latest code

Reply:
hi @meomancer it's not yet fixed, here's from develop branch: https://github.com/healthsites/healthsites/blob/bd2fe2bb7f4b5105b9a2292605532229a91135b1/django_project/social_users/templates/social_users/profile.html#L23